redirects/aei/ditamaps/index.html

```diff
@@ -0,0 +1 @@
+/content-services/latest
```

Comment:
I guess you are right here but that is in a ditamaps directory? :-) !

Reply:
Most of the other redirects just point to the top-level index page, so I'm following the same standard. Plus, the old ditamaps were internal files, so this shouldn't affect the outcome (famous last words)! ;-)